src/encoded/schemas/matrix_file.json

```diff
@@ -271,6 +271,48 @@
                     }
                 }
             }
+        },
+        "cell_mapping": {
+            "title": "Cell mapping information for this matrix",
```

Comment:
just reiterating the property name is standard. 'Cell mapping'
no real rhyme or reason, but if this is displayed in a report column, etc. then the long name could be too much